aries-site/src/themes/aries.js

```diff
@@ -250,15 +395,41 @@ export const aries = {
     },
   },
   text: {
-    small: fontSizing(0), // 18
-    medium: fontSizing(1), // 24
-    large: fontSizing(3), // 36
-  },
-  paragraph: {
-    small: fontSizing(0),
-    medium: fontSizing(1),
-    large: fontSizing(3),
+    xsmall: {
+      size: '8.4px',
+      height: '14.4px',
+      maxWidth: '201.6px',
+    },
+    small: {
```

Comment:
Noticing this size of "11.6" doesn't match what was previously defined as "18" -- when I run the size, the text isn't coming through at the right values. Here is what a home tile is looking like. I am fine with hard-coding but let's make sure the values match what the previous calculations mapped to (unless there has been rationale to shift these values)

<img width="494" alt="Screen Shot 2020-02-05 at 10 38 57 AM" src="https://user-images.githubusercontent.com/12522275/73872054-d68eb580-4803-11ea-86f2-2c5d78a3e803.png">

Reply:
Finalizing Text/HEading/Paragraph sizes is currently work in progress, thanks for raising it. I'll keep this PR updated with upcoming changes.